package/packer/vmware/vmware.json

```diff
@@ -0,0 +1,26 @@
+{
+  "variables": {
+    "password": "rancher"
+  },
+  "builders": [{
+    "type": "vmware-iso",
+    "boot_wait": "30s",
+    "boot_command": ["rancher","<enter>","wget --no-check-certificate https://raw.githubusercontent.com/wjimenez5271/k3os-vagrant/master/cloud-config.yml","<enter>","sudo os-config","<enter>","1","<enter>","1","<enter>","y","<enter>","/home/rancher/cloud-config.yml","<enter>","N","<enter>","{{user `password`}}","<enter>","{{user `password`}}","<enter>","1","<enter>","<enter>"],
```

Comment:
Pulling from a users personal gist is a bit scary to me, no offense! This should be pulled either from the rancher repo or gist.